Lib/fontTools/feaLib/parser.py

```diff
@@ -1434,6 +1517,13 @@ def expect_decipoint_(self):
             raise FeatureLibError("Expected an integer or floating-point number",
                                   self.cur_token_location_)
 
+    def expect_decimal_or_hexadecimal_(self):
```

Comment:
how is this `expect_decimal_or_hexadecimal` any different from the existing `expect_number_` (apart from the error message)? Is that what you wanted to change? Then perhaps the lexer needs to be able to distinguish between decimal and hexadecimal numbers.

Reply:
The info message is different. Using the one for numbers didn't seem appropriate as it could lead to user confusion.

Reply:
I see.. but then does it mean we are implicitly allowing hexadecimal digits elsewhere, where the spec only mention decimal? that could be problematic. If the spec only allows hexadecimal numbers in some places (like in cv feature) then the lexer needs to be able to distinguish between these.

Reply:
good point. Should I try to address that in this PR, or sort that out in a later one?

Reply:
later maybe, thanks

Reply:
you can add a TODO and fix it later